README.md

```diff
@@ -28,6 +28,13 @@ This is a list of the Cargo features available for mbedtls. Features in
                         will result in undefined instruction exceptions on
                         unsupported processors. On SGX, this feature is
                         enabled automatically.
+* *mpi_force_c_code* MbedTLS uses assembly code for MPI functions, when available.
```

Comment:
This description belongs in the `mbedtls-sys` feature list. Here you can say that this enables the sys feature.